CONTRIBUTING.md

```diff
@@ -287,6 +287,66 @@ If the SVG is sourced from:
 
 In general, make sure the URL does not contain any tracking identifiers.
 
+#### Aliases
+
+Lastly, we aim to provide aliases of three types for various reasons. Each type of alias and their motivation can be found below. If you're unsure, you can mention an alias you're considering in your Pull Request so it can be discussed.
+
+##### Also Known As
+
+We collect "also known as" names to make it easier to find brands that are known by different names or by their abbreviation/full name. To add an "also known as" name you add the following to the icon data:
+
+```json
+{
+  "title": "the original title",
+  "aliases": {
+    "aka": [
+      "tot"
```

Comment:
Maybe adding another example here to increase readability? This array looks to me more like a configuration file :sweat_smile: 

```suggestion
      "tot",
      "thetitle"
```